h/search/index.py

```diff
@@ -107,28 +107,31 @@ def __init__(self, session, es_client, request, target_index=None, op_type='inde
         else:
             self._target_index = target_index
 
-    def index(self, annotation_ids=None):
+    def index(self, annotation_ids=None, windowsize=PG_WINDOW_SIZE, chunk_size=ES_CHUNK_SIZE):
         """
         Reindex annotations.
 
         :param annotation_ids: a list of ids to reindex, reindexes all when `None`.
         :type annotation_ids: collection
+        :param windowsize: how often to log indexing progress
```

Comment:
I think this docstring could be clearer about the meaning of `windowsize`. It means the number of annotations to index in between progress log statements?